infra/tf/README.md

```diff
@@ -7,4 +7,35 @@ Terraform is our tool of choice for automating our 'cloud infrastructure'. In pa
 ## Contents
 
 * Something about Truss Terraform Modules
-* How to layout/structure a Trraform Project
+* How to layout/structure a Terraform Project
+
+### Starting a module experiment
+
+...
+
+### Going public
+
+When you're ready to turn the prototype module into a published one, there's a few things you'll need to do if you haven't already:
+
+1. 🔒 Add your repo to the [list of repos](https://github.com/trussworks/truss-infra/blob/master/modules/github-truss-repository/main.tf) managed by our GitHub repo management module.
```

Comment:
This link isn't to a list of repos, is it the link you intended?

Reply:
good point. I'll update to `https://github.com/trussworks/truss-infra/blob/master/truss/github/main.tf` in the next commit